### ISSUE TITLE
Scale down & cache thumbnail images

### ISSUE THREAD

**Issue by scooper4711** [J-Guenther/foundryvtt-journal-thumbnail#4]
Having enabled this module, I see in the network traffic that all images for all journal entries are downloaded upon initial load of the system.

When images are large, this can lead to a considerable delay in initial load time for the user.

My suggestion is to resize the images on the server and cache the images. This will allow for all journal entries to have thumbnails without adding megabytes to the initial download screen.

**Comment by J-Guenther** [comment]
You are right about this issue. I'ver never paid attention to that kind of problem. Your suggestion sounds good. I will take a look into it and investigate this further.

**Comment by scooper4711** [comment]
I love your module but alas had to disable it because I have some big images - even after converting them to webp. 

One challenge you will have with my suggestion is updating the thumbnail if the image is changed.
